content_schemas/dist/formats/substitute/notification/schema.json

```diff
@@ -2,29 +2,13 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "required": [
-    "analytics_identifier",
```

Comment:
Maybe this commit could be merged into [`Add schema for new substitute type`](https://github.com/alphagov/publishing-api/pull/2664/commits/0659f302cc25e604d66f7ca292c2cbc958fc59a8), so these lines are never added in the first place?

Reply:
Done as above 👍